internal/processing/processors/v2alpha1/reconciliation.go

```diff
@@ -54,3 +54,22 @@ func NewReconciliation(apiRuleV2alpha1 *gatewayv2alpha1.APIRule, apiRuleV1beta1
 		config:          config,
 	}
 }
+
+func filterDuplicatePaths(rules []gatewayv2alpha1.Rule) []gatewayv2alpha1.Rule {
+	uniqueRules := make(map[string]gatewayv2alpha1.Rule)
+	for _, rule := range rules {
+		uniqueRules[rule.Path] = rule
+	}
+	var uniqueRulesSlice []gatewayv2alpha1.Rule
+	for _, rule := range uniqueRules {
+		uniqueRulesSlice = append(uniqueRulesSlice, rule)
+	}
+	return uniqueRulesSlice
+}
+
+func findServiceNamespace(api *gatewayv2alpha1.APIRule, rule *gatewayv2alpha1.Rule) string {
```

Comment:
The implementation changed to what is used in istio VS processor: https://github.com/kyma-project/api-gateway/blob/371e92079db3be78beaad662638799e29871fd6a/internal/helpers/service.go#L15.

Reply:
OK, it should have stayed :D